tools/bumper/component_commands_test.go

```diff
@@ -460,4 +460,43 @@ var _ = Describe("Testing internal git", func() {
 			isValid:               false,
 		}),
 	)
+
+	type resetRepoParams struct {
+		files []string
+	}
+
+	DescribeTable("resetRepo function",
+		func(r resetRepoParams) {
+			defer os.RemoveAll(gitComponent.gitRepo.localDir)
+			worktree, err := gitComponent.gitRepo.repo.Worktree()
+
+			By("Modifying files in the Repo")
+			modifyFiles(gitComponent.gitRepo.localDir, r.files)
+			status, err := worktree.Status()
+			Expect(err).ToNot(HaveOccurred(), "should succeed to get repo status")
+			isFilesAdded := len(r.files) != 0
+			Expect(status.IsClean()).To(Equal(!isFilesAdded), "repo should have expected modified files")
```

Comment:
This expect is hard to understand, if we're checking that there are newly modified files, it would be more readable to expect
`!status.IsClean()` and `isFilesAdded`

Reply:
do you mean this?
```
Expect(!status.IsClean()).To(Equal(isFilesAdded), "repo should have expected modified files")
```

Reply:
Yes

Reply:
Done